content/rule_book/quiz_events.md

```diff
@@ -1,117 +1,113 @@
-- It does not fulfill all requirements specific to the Question Type.
-- Three Incorrect answers by a quizzer eliminate that quizzer from the quiz. The quizzer must leave the platform immediately.
-    - Incorrect answers on bonus questions do not count towards this total.
+- Incorrect information has been given
+- The information given within the time limit is incomplete
+- The response does not fulfill all requirements specific to the question type
 
-#### Example
+**Error**
+: An incorrect answer is also known as an error
 
-Incorrect Information, e.g. If the wrong person (i.e. Father, Son, Holy Spirit) in the deity (God) is given.
+**Err-Out**
+: 3 errors by the same quizzer in the same quiz
 
-All requirements are not fulfilled, Any word is omitted or added in a Finish-The-Verse, Finish-This, or Quote Question or the wrong verse is begun.
+An err-out eliminates that quizzer from the quiz. The quizzer must leave the platform immediately.
 
 ## Challenges
 
-Challenges exist so quizzers can contest rulings made by quiz officials.
-
-The team captain or co-captain may challenge if:
+Challenges exist so quizzers can contest rulings made by quiz officials. The team captain or co-captain may challenge if:
 
 - There is reason to believe the quiz officials have applied the rule book incorrectly
-- There has been a mispronunciation by the quizmaster
-- There has been an invalid question
-- The next question has not been called
-    - A challenge must be made at the time of the quizmaster's decision and before the next question is called. If a timeout is called immediately after a ruling, and a quizzer stands to challenge, the challenge may be done prior to the timeout.
-- No conferring has occurred between the quizzer challenging and anyone else. Conferring with anyone else prior to a challenge will result in:
-    - The challenge being disallowed
-    - A foul being awarded to the quizzer.
+- The quizmaster mispronounced a word
+- The question is invalid
+
+A challenge must be made at the time of the quizmaster's decision and before the next question's type is made known. If a timeout is called immediately after a ruling, and a quizzer stands to challenge, the challenge may be conducted prior to the timeout.
+
+No conferring is allowed between the quizzer challenging and anyone else. Conferring with anyone else prior to a challenge will result in:
 
-Once a team has requested an explanation of a ruling, and that explanation has been given, all teams lose the right to challenge.
+- The challenge being disallowed
+- A foul being awarded to the quizzer
 
-- If an explanation of a ruling is requested, the quizmaster will ask if any team wants to challenge.
+Once a team has requested an explanation of a ruling, and that explanation has been given, all teams lose the right to challenge. If an explanation of a ruling is requested, the quizmaster will ask if any team wants to challenge.
 
 The result of a challenge may not be challenged.
 
```

Comment:
I see this in the HTML but not here. 
![Screen Shot 2020-11-25 at 8 19 54 PM](https://user-images.githubusercontent.com/13351944/100297000-a44d7080-2f5b-11eb-824d-02a21aabdfa9.png)

it's not just if the question "is ruled illegal"   It's just "if the outcome of the challenge dictates the question needs to be redone"

Reply:
0d6e1ba